### ISSUE TITLE
Object Null Exception if rel is not set on stylesheet link

### ISSUE THREAD

**Issue by richard-scryber** [richard-scryber/scryber.core#42]
```
<link href='./HTML/CSS/HelloPrint.css' 
            media='print' />
```

Will throw a really bad object reference not set to an instance of object.

```
<link rel='stylesheet'
            href='./HTML/CSS/HelloPrint.css' 
            media='print' />
```

Works

**Comment by richard-scryber** [comment]
The 'rel'ationship attribute is required. Update the exception to be more descriptive, rather than assume it is a stylesheet.

**Comment by richard-scryber** [comment]
Fixed in 5.0.6.3 to log an error, or raise an exception if we are in strict conformance mode.
